### DISCUSSION TITLE
THR phosphorylated

### DISCUSSION THREAD

**Discussion by alexmcff** [Electrostatics/pdb2pqr#309] [Q&A]
Hello,

I am trying to use the pdb2pqr server with a pdb that has a phosphorylated threonine. 
Has anyone experience in doing this?
I would appreciate any help :-)

**Comment by jbardhan** [comment]
Hi,

David Huggins published work several years ago that provided details. I don't know whether their choices will apply to yours, but it may be a starting point: 

 doi:10.1371/journal.pcbi.1000880

Best regards,
Jay

**Reply by alexmcff** [reply]
Thank you!
I will check it out

best regards

Cristina
